README.md

```diff
@@ -11,15 +11,20 @@ For a detailed description of the driver, including a step by step deployment ex
 
 Prerequisite
 ------------
+
+A NIOS DDI Appliance with cloud automation License.
+
 To use the plugin, you need access to the Infoblox DDI product. For evaluation purposes, you can download a
 virtual version of the product from the [Infoblox Download Center](https://www.infoblox.com/infoblox-download-center).
```

Comment:
and you can also assign temp license by login into the Infoblox DDI appliance console with this command ```set temp_license```

Reply:
added

Reply:
done